code/modules/power/cable.dm

```diff
@@ -23,13 +23,28 @@ GLOBAL_LIST_INIT(wire_node_generating_types, typecacheof(list(/obj/structure/gri
 	var/linked_dirs = 0 //bitflag
 	var/node = FALSE //used for sprites display
 	var/datum/powernet/powernet
+	color = "#FFF200"
+
+/obj/structure/cable/red
+	color = "#DA0000"

```

Comment:
These should probably use the color defines since they are used for something that actually matters.